packages/block-library/src/classic/edit.js

```diff
@@ -141,6 +141,11 @@ export default class ClassicEdit extends Component {
 		editor.on(
 			'Paste Change input Undo Redo',
 			debounce( () => {
+				// We need this check because when we remove the editor (onUnmount)
+				// due to the usage of `debounce`, this callback is executed in
+				// another tick. This results in setting the content to empty.
+				if ( editor._hasBeenRemoved ) return;
```

Comment:
Could we use `cancel`? https://lodash.com/docs/#debounce

Reply:
Yes. Actually I had considered at first ( before deciding to utilise editor's `onRemove` event ), so in my mind was too ugly to implement outside.. That said, since we are keeping lodash's `debounce` it's the way to go.

Thanks @ellatrix!